_components/form-templates/02-address-form.md

```diff
@@ -33,6 +33,13 @@ lead: Enter a standard U.S. mailing or shipping address
     <ul class="usa-content-list">
       <li>As you customize this form template, make sure it continues to follow the <a href="{{ site.baseurl }}/form-templates/">accessibility guidelines for form templates</a> and the <a href="{{ site.baseurl }}/form-controls/">accessibility guidelines for form controls</a>.</li>
     </ul>
+    <ul class="usa-content-list">
+      <li>As you customize this form template, make sure it continues to follow the <a href="{{ site.baseurl }}/form-templates/">accessibility guidelines for form templates</a> and the <a href="{{ site.baseurl }}/form-controls/">accessibility guidelines for form controls</a>.</li>
+    </ul>
+    <h4>Additional Information</h4>
+    <p>
+      <a href="https://pe.usps.com/text/pub28/28c1_001.htm">USPS Publication 28</a> defines the mailing standards in the United States (see <a href="https://drive.google.com/file/d/1T1dmOfuBoOypUw4FbOSYtOHaTGUGE9MC/view?usp=sharing">this summary of applicable standards</a> 🔒 for easier reference and search)
```

Comment:
Remove "(see <a href="https://drive.google.com/file/d/1T1dmOfuBoOypUw4FbOSYtOHaTGUGE9MC/view?usp=sharing">this summary of applicable standards</a> 🔒 for easier reference and search)"